Tests/Tests.cs

```diff
@@ -474,8 +470,8 @@ public void SwapValuesEx3Sln()
         {
             var meter1 = "Meter001";
             var meter2 = "Meter002";
-            var startDateRelative = "*-5m";
-            var endDateRelative = "*";
+            var startDateRelative = "*-6m";
```

Comment:
just out of curiosity, why did we change these?

Reply:
This was changed because I think having the ```endDateRelative``` as * was causing the tests to fail. My hypothesis is that data was getting swapped as new data was being written in.